pkgs/by-name/qo/qodana-cli/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+}:
+
+let
+  platforms = {
+    x86_64-linux = {
+      filename = "qodana_linux_x86_64";
+      hash = "sha256-5dMwLlwA1r3A9TD05IUoBbiKvGXzSUO8QqYb0z4L1cU=";
+    };
+    x86_64-darwin = {
+      filename = "qodana_darwin_x86_64";
+      hash = "sha256-j8vrpoAm8zoNJlsGS7tgfobId11Omq3gyr8bJiCtYBU=";
+    };
+    aarch64-linux = {
+      filename = "qodana_linux_arm64";
+      hash = "sha256-8bi6G+XcRTuofeT3XF+G5ZRFuxPeQH+AnORHWgSxpw0=";
+    };
+    aarch64-darwin = {
+      filename = "qodana_darwin_arm64";
+      hash = "sha256-uAFn9aT/LAVGcbmFqx3MTVnRSchv0RVZNVQ61NBceqc=";
+    };
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  version = "2024.1.8";
```

```suggestion
  version = "2024.2.5";
```
New version released by now